src/sinks/influxdb_metrics.rs

```diff
@@ -219,11 +225,10 @@ impl InfluxDBSvc {
 
         let request = hyper::Request::get(uri).body(hyper::Body::empty()).unwrap();
 
-        let https = HttpsConnector::new(4).expect("TLS initialization failed");
-        let client = hyper::Client::builder().build(https);
+        let mut client = HttpClient::new(resolver, None)?;
```

Comment:
Do we not support the usual TLS options here?

Reply:
Looks like it, I didn't want to change the behavior this just consolidates all tls building to one part of the codebase.